src/app/data-planes/components/DataPlaneList.vue

```diff
@@ -1,114 +1,141 @@
 <template>
-  <ContentWrapper>
-    <template #content>
-      <DataOverview
-        :selected-entity-name="dataPlaneOverview?.name"
-        :page-size="PAGE_SIZE_DEFAULT"
-        :is-loading="props.isLoading"
-        :error="error"
-        :empty-state="EMPTY_STATE"
-        :table-data="filteredTableData"
-        :table-data-is-empty="filteredTableData.data.length === 0"
-        :next="props.nextUrl !== null"
-        :page-offset="props.pageOffset"
-        @table-action="loadEntity"
-        @load-data="loadData"
+  <AppCollection
+    data-testid="data-plane-collection"
+    :empty-state-title="t('common.emptyState.title')"
+    :empty-state-message="t('common.emptyState.message', { type: props.gateways ? 'Gateways' : 'Data plane proxies' })"
+    :headers="[
+      { label: 'Name', key: 'name' },
+      { label: 'Status', key: 'status' },
+      props.gateways ? { label: 'Type', key: 'type' } : undefined,
+      { label: 'Service', key: 'service' },
+      !props.gateways ? { label: 'Protocol', key: 'protocol' } : undefined,
+      isMultiZoneMode ? { label: 'Zone', key: 'zone' } : undefined,
+      { label: 'Last Updated', key: 'lastUpdated' },
+      { label: 'Kuma DP version', key: 'dpVersion' },
+      { label: 'Actions', key: 'actions', hideLabel: true },
+    ].filter(Boolean)"
+    :page-number="props.pageNumber"
+    :page-size="props.pageSize"
+    :total="props.total"
+    :items="props.items ? transformToTableData(props.items) : []"
+    :error="props.error"
+    @change="(val: ChangeValue) => emit('change', val)"
```

Comment:
Hmm let me look at this, I would also like to avoid needing the type parameter here seeing as we don't access the data at all. Little cautious about the weird global $event thing though (thats what it is is it?). Let me take a look at this one though.